right-for-life_front/src/components/Header/Header.jsx

```diff
@@ -9,12 +10,38 @@ export const Header = () => {
 					<li className="mx-4 cursor-pointer ">
 						<Logo />
 					</li>
-					<li className="mx-4 cursor-pointer hover:text-gray-700">Новости</li>
-					<li className="mx-4 cursor-pointer hover:text-gray-700">Питомцы</li>
-					<li className="mx-4 cursor-pointer hover:text-gray-700">Товары</li>
-					<li className="mx-4 cursor-pointer hover:text-gray-700">
+					<NavLink
+						className="mx-4 cursor-pointer hover:text-gray-700"
```

Comment:
[optional] To avoid code duplication you might want to extract repetitive set of utilities into custom component class with @apply directive, see https://tailwindcss.com/docs/extracting-components/#extracting-css-components-with-apply